db/migrate/20231025082805_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :title
+      t.string :text
+      t.integer :commentsCounter
```

Comment:
- [ ]  Column holding the foreign key to the `users` table should be named `author_id` in the `posts` table as requested in the project requirements, Please make sure to fix this. It should be like this:
```ruby 
t.references :author, null: false, foreign_key: {to_table: 'users'}, index: true
```

![image](https://github.com/wickathou/mcv-blog/assets/83803414/9c27e954-0008-4361-836b-a6026a742fd9)